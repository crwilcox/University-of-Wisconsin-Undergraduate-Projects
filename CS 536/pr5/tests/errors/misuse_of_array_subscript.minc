void main()
{
	//Attempt to use an array subscript on a non-array. (Pointers are not arrays.)
	int myInteger = 5;
	int someOtherInteger = myInteger[0];

	int* myPointer = &myInteger;
	int accessInt = myPointer[0];
}
