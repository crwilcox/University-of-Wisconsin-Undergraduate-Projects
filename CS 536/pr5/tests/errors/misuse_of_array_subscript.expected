5:34:Using [] on non-array
8:27:Using [] on non-array
8:16:Type mismatch
